src/teleport.js

```diff
@@ -0,0 +1,35 @@
+import { checkForAlpine } from './utils'
+
+const AlpineTeleportMagicMethod = {
+    start() {
+        checkForAlpine()
+        Alpine.addMagicProperty('teleport', ($el) => {
+            return function (destination = document.body, $element = $el, prepend = false) {
+                // find out destination
```

Comment:
I think we will want to follow a pattern on adding options as an object, especially when it's a boolean. So it would be `$teleport(from, to, options)`in this case

Reply:
Yeah, i think we can improve this part. Thanks for the suggestion.